sdk/monitor/monitor-opentelemetry-exporter/test/internal/functional/metric.test.ts

```diff
@@ -35,8 +35,10 @@ describe("Metric Exporter Scenarios", () => {
       await scenario.run();
       // promisify doesn't work on this, so use callbacks/done for now
       await scenario.flush();
-      assertMetricExpectation(ingest, scenario.expectation);
-      assertCount(ingest, scenario.expectation);
+      setTimeout(() => {
+        assertMetricExpectation(ingest, scenario.expectation);
+        assertCount(ingest, scenario.expectation);
+      }, 10);
```

Comment:
nitpick: rewriting it for simplicity:
```suggestion
      await delay(10); // imported from @azure/core-util
      assertMetricExpectation(ingest, scenario.expectation);
      assertCount(ingest, scenario.expectation);
```

Reply:
Tested locally and `await delay(x)` fails as the test doesn't seem to respect the await in the same way as `await scenario.flush()`.